Casks/f/flixtools.rb

```diff
@@ -8,8 +8,14 @@
   homepage "https://www.flixtools.com/"
 
   livecheck do
```

Comment:
```suggestion
  # The version text on the download page (https://www.flixtools.com/download/)
  # isn't present in the HTML and is rendered with JavaScript using data from
  # the `versionchk.json` file.
  livecheck do
```

Per Bevan's suggestion, here's a suggested comment (in line with similar ones in other casks).